src/js/common/components/form-elements/ExpandingGroup.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+
+export default function ExpandingGroup({ children, open, additionalClass }) {
```

Comment:
Is it worth adding comments above to explain what each of these props are? We've done that in some other components, and I think for this one it might be particularly useful to explain what `children[0]` should be and what `children[1]` and greater should be, since, if I'm reading this right, they have specific purposes (`children[0]` being the question that determines whether or not more questions should be expanded?).